dotCMS/src/integration-test/java/com/dotmarketing/portlets/containers/business/ContainerStructureFinderStrategyResolverTest.java

```diff
@@ -56,4 +72,81 @@ public void get_container_structure_from_file_name_() throws Exception {
+     */
+    @Test
+    public void test_default_layout() throws Exception {
+
+        //Getting the current default host
+        final String pathRoot             = "/application/containers/test"+System.currentTimeMillis();
+        final Host currentDefaultHost     = APILocator.getHostAPI().findDefaultHost(APILocator.systemUser(), false);
+        final Folder containerFolder      = this.createAppContainerTest(currentDefaultHost, pathRoot);
+        final String contentTypeVariable  = "testfa"+System.currentTimeMillis();
+        ContentType fileAssetContentType  = ImmutableFileAssetContentType.builder()
+                .name("FileAssetTestContentType").variable(contentTypeVariable).build();
+        fileAssetContentType              = APILocator.getContentTypeAPI(APILocator.systemUser()).save(fileAssetContentType);
+        final List<FileAsset> assets      = Arrays.asList(
+                this.createFileAsset("container", ".vtl",
+                        "$dotJSON.put(\"title\", \"Test No VTL and All Content Types\")\n" +
+                                "$dotJSON.put(\"max_contentlets\", 25)\n" +
+                                "$dotJSON.put(\"useDefaultLayout\",\"*\")", containerFolder, fileAssetContentType),
+                this.createFileAsset("default_container", ".vtl", "$title", containerFolder, fileAssetContentType));
+
+        final ContainerStructureFinderStrategyResolver containerStructureFinderStrategyResolver =
+                new ContainerStructureFinderStrategyResolver();
+
+        final Container container = APILocator.getContainerAPI()
+                .getWorkingContainerByFolderPath(pathRoot, currentDefaultHost, APILocator.systemUser(), false);
+        final Optional<ContainerStructureFinderStrategy> containerStructureFinderStrategy =
+                containerStructureFinderStrategyResolver.get(container);
+
+        Assert.assertTrue(containerStructureFinderStrategy.isPresent());
+
+        final  List<ContainerStructure> containerStructures = containerStructureFinderStrategy.get().apply(container);
+
+        Assert.assertNotNull(containerStructures);
+        Assert.assertFalse(containerStructures.isEmpty());
+
+        final List<ContentType> contentTypes = APILocator.getContentTypeAPI(APILocator.systemUser()).findAll();
+        for (final ContentType contentType : contentTypes) {
+
+            Assert.assertTrue("Content Type: " + contentType.variable() + " does not included on the container",
+                    containerStructures.stream().anyMatch(containerStructure -> containerStructure.getStructureId().equals(contentType.id())));
+        }
+    }
+
+    private FileAsset createFileAsset (final String fileName1,
+                                       final String fileExtension,
+                                       final String fileContent,
+                                       final Folder root1,
+                                       final ContentType fileAssetContentType) throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6347242)